src/main/java/com/verygood/security/coding/util/CipherUtils.java

```diff
@@ -0,0 +1,70 @@
+package com.verygood.security.coding.util;
+
+import com.verygood.security.coding.exception.CipherDecryptException;
+import com.verygood.security.coding.exception.CipherEncryptException;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.SecretKeySpec;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Arrays;
+import java.util.Base64;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.util.Objects.requireNonNull;
+
+@Slf4j
+public enum CipherUtils {
+    ;
+    private static final String DEFAULT_ALGORITHM = "AES/ECB/PKCS5Padding";
+    private static SecretKeySpec secretKey;
+
+    public static String encrypt(String strToEncrypt, String secret) {
+        return encrypt(strToEncrypt, secret, DEFAULT_ALGORITHM);
+    }
+
+    public static String encrypt(String whatToEncrypt, String secret, String algorithm) throws CipherEncryptException {
```

Comment:
Looks like `encrypt` and `decrypt` logic can be in one method not to duplicate that much code.